webapp/components/self-navigator.html

```diff
@@ -15,6 +15,10 @@
             type: String,
             computed: 'encodeTestPath(path)'
           },
+          isSubfolder: {
+            type: Boolean,
+            computed: '_computeIsSubfolder(path)',
```

Comment:
Nit: I don't think the leading underscore is used anywhere else in this project.